src/profile.c

```diff
@@ -175,9 +177,12 @@ void HookedLineOutput(Obj func, char type)
               (int)profileState.lastNotOutputted.fileID);
     }
 
-    fprintf(profileState.Stream,
-            "{\"Type\":\"%c\",\"Fun\":\"%s\",\"Line\":%d,\"EndLine\":%d,\"File\":\"%s\"}\n",
-            type, name_c, (int)startline, (int)endline, filename_c);
+    // We output 'File' here for backwards compatibility
```

Comment:
"backwards compatibility" with what? Older versions of the `profiling` package that need / expect this?

Perhaps also explain what the replacement is? Perhaps like this?
> Output  information on the function into the JSON stream. Note that the 'File' member is only printed for backwards compatibility with older versions of the profiling package; newer versions use the 'FileId' member instead.

Bonus points for saying which versions of profiling use 'FileId' (and do they still accept older files with 'File' but not 'FileId'?). Of course this may not yet be possible, as presumably no such version of profiling has yet been released, but perhaps you can simply decide on the planned version now and put it in anyway?

Reply:
Can do.